src/patterns.rst

```diff
@@ -61,6 +61,17 @@ a possibly nested :t:`identifier pattern` of a :t:`slice pattern`\ ``.``
 The :t:`expected type` of a :t:`pattern` is the :t:`type` of the :t:`value` the
 :t:`pattern` is being matched against.
 
+:dp:`fls_TUanRT7WU14E`
+It is a static error when :t:`[lexical element]s` match multiple alternations of a
+:s:`PatternWithoutRange` ambiguously, except for the following case:
+
+.. syntax::
+
+   $$&$$ $$mut$$ Identifier
+
+This ambiguous case is interpreted as a :t:`reference pattern` with
+:t:`keyword` ``mut`` containing an :t:`identifier pattern`.
+
```

Comment:
How about something along the lines of:

It is a static error when lexical elements match multiple alternations of PatternWithoutRange, except when the pattern is &mut identifier. Such a pattern is interpreted as a reference pattern with keyword mut containing an identifier pattern.

Note that I dropped "ambiguously" and the syntax section.